pkg/server/api/http.go

```diff
@@ -42,27 +49,43 @@ type managers struct {
 	crt *mgrcrt.CertManager
 }
 
-type HTTPServer struct {
+type Server struct {
```

Comment:
Maybe the filename should also be renamed?
Or move the grpc part to another file?

Reply:
Renamed.